config/image/enterprise/kustomization.yaml

```diff
@@ -6,7 +6,7 @@ kind: Component
 images:
 - name: kong
   newName: kong/kong-gateway
-  newTag: '3.0'
+  newTag: '3.1'
 - name: kong-placeholder
   newName: kong/kong-gateway
   newTag: '3.0'
```

Comment:
should here also be updated to `3.1`?

Reply:
I believe so, yes. Adding a change with this to this PR